catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -109,7 +107,6 @@ public AdminPollerServiceBean(ConfigurationAdmin configurationAdmin, org.osgi.se
 
     public void init() {
         serviceFactoryFilter = getServiceFactoryFilterProperties();
-        serviceFilter = getServiceFilterProperties();
```

Comment:
You can probably do a quick search to see if it's used anywhere in DDF and in downstream projects. I'd think that it'd be okay to remove.